lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2381,38 +2422,80 @@ private static function get_block_nodes( $theme_json, $selectors = array() ) {
 		}
 
 		foreach ( $theme_json['styles']['blocks'] as $name => $node ) {
-			$selector = null;
-			if ( isset( $selectors[ $name ]['selector'] ) ) {
-				$selector = $selectors[ $name ]['selector'];
-			}
+			$selector            = $selectors[ $name ]['selector'] ?? null;
```

Comment:
I'm afraid the null coalescing operator isn't officially supported in WP yet. There's a ticket for it [here](https://core.trac.wordpress.org/ticket/58874).

Reply:
Thanks for flagging that. I had a suspicion that was the case although I recalled some recent performance PRs swapping out `_wp_array_get` for it. It's used a fair bit now already within this class.

Do you think it is that much of a nuisance when it comes to backporting?

Reply:
Yeahhh those perf improvements were changed before landing in core - see the discussion [here](https://github.com/WordPress/wordpress-develop/pull/5244). I'd avoid it if you don't want the backport PR to turn into an argument 😂